Cargo.toml

```diff
@@ -18,7 +18,7 @@ hmac = "0.7"
 sha2 = "0.8"
 serde_derive = "~1.0.82"
 rand = { version = "0.6", features = ["wasm-bindgen"] }
-recrypt = "0.6"
+recrypt = "^0.6.2"
```

Comment:
I'm not sure this code change is necessary really? This will get picked up the next time we make a change here right? Did you think we'd publish a new version of this binding with this change?

Reply:
Maybe it's worthwhile to publish this out so we can say that our SDKs are using audited code?

Reply:
To me I think it's worth it because getting the audited code as the floor.